jerry-core/vm/vm.c

```diff
@@ -1984,6 +1966,24 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
           *stack_top_p++ = result;
           continue;
         }
+        case VM_OC_RESOLVE_BASE_FOR_CALL:
+        {
+          ecma_value_t this_value = stack_top_p[-3];
```

Comment:
When this happens? Before the arguments are pushed or after? If after, the -3 will be probably incorrect.

Reply:
It happens before the arguments are pushed.